ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -117,25 +110,25 @@ private fun PurchaseButton(state: PaywallViewState.Template2, viewModel: Paywall
         verticalArrangement = Arrangement.Center,
     ) {
         val activity = LocalContext.current.getActivity() ?: error("Error finding activity")
```

Comment:
Not in this PR, but I just realized that this is coupling it with an `Activity`, which fails when loading the preview (#1239).

Reply:
Added a log in https://github.com/RevenueCat/purchases-android/pull/1242/commits/ac5858a4cf8eeeecd419b0221c64b9ed408a3b2d